platform-enterprise_versioned_sidebars/version-22.4-sidebars.json

```diff
@@ -1,6 +1,6 @@
 {
   "platformSidebar": [
-    "platform-enterprise",
+    "index",
```

Comment:
By moving to index here we get rid of the `/platform-enterprise/platform-enterpise` shenanigans.